modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/input_move_boxes.js

```diff
@@ -283,10 +289,16 @@ AUI.add(
 						var btnDisabledRight = true;
 
 						if (target.get('length') > 0) {
-							if (target == instance._rightBox) {
+							if (
+								target == instance._rightBox &&
```

Comment:
Let's go with `===` whenever we can.

Reply:
Ouch, didn't see this, but will fix it later... unless @markocikos beats me to it 😉